starter/src/test/java/com/starter/fullstack/rest/InventoryControllerTest.java

```diff
@@ -0,0 +1,87 @@
+// TASK 2: Created a new Java file to
+package com.starter.fullstack.rest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.starter.fullstack.api.Inventory;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@RunWith(SpringRunner.class)
+public class InventoryControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private MongoTemplate mongoTemplate;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    private Inventory inventory;
+
+    @Before
+    public void setup() throws Throwable {
+        this.inventory = new Inventory();
+        this.inventory.setId("ID");
```

Comment:
You have some magic strings here. Even though it is a test file, you will always want to eliminate magic strings. If you are unsure what magic strings are checkout the link below.
https://dev.to/ruben_alapont/magic-numbers-and-magic-strings-its-time-to-talk-about-it-ci2

Reply:
Created constants and replaced the magic strings
![Screenshot 2024-07-09 at 18 59 07](https://github.com/K-Shields1994/fullstack-starter/assets/120108762/315083d4-89e0-4d02-9617-91879ce088e4)